crate/operator/data/cratedb_cluster_last_user_activity-collector.yaml

```diff
@@ -0,0 +1,14 @@
+---
+collector_name: cratedb_cluster_last_user_activity_collector
+# This metric is intended to know if the cluster is used
+# and to calculate for how long it has not been used.
+metrics:
+- help: "Indicates when the last job occured on the cluster."
+  metric_name: cratedb_cluster_last_user_activity
+  query: |
+    SELECT max(ended) AS cratedb_cluster_last_user_activity
+    FROM sys.jobs_log
+    WHERE username NOT IN ('crate', 'system');
+  type: gauge
+  value_label: cratedb_cluster_last_user_activity
+  values: [cratedb_cluster_last_user_activity]
```

Comment:
No string opinion here, but maybe use something like `cratedb_clusters_user_last_seen` (I think most other metrics use `clusters`.

Reply:
I kind of like activity as it's more specific, in this case. Indicates that the user has actually done something, rather than was just "seen" (i.e. logged in)

Reply:
We already have a "last_seen" column in the cluster table, it might be confusing to use `last_seen" for that too.